### DISCUSSION TITLE
블로그 내 포스트를 클릭 시 새 창에서 열리는 것

### DISCUSSION THREAD

**Discussion by dameun0527** [pronist/hELLO#254] [Q&A] [answered]
안녕하세요. 만들어주신 스킨을 잘 쓰고 있습니다!
혹시 블로그 내에서 내가 쓴 포스트를 클릭했을 때 새로운 창에서 열리지 않고 기존 창에서 열리게 하는 방법은 없을까요?

**Comment by pronist** [comment]
@dameun0527 안녕하세요.

기존창에서 열리게 하는 방법이 있기는 하지만, 코드가 중복되는 부분이 많아 **skin.html** 에서는 수정하기 어렵습니다.
원본 소스코드에서는 `target='_blank'` 한줄만 지워도 됩니다.

https://github.com/pronist/hello/blob/f73839ad9851d3c82e4d9864790805ff39df2510/assets/pug/Mixins/Index.pug#L17-L19

티스토리에서는 리스트에 블로그내 링크가 아닌 
외부링크도 사용할 수 있기 때문에 새 탭이 열리도록 구성한 것입니다.

**Reply by dameun0527** [reply]
답변 감사합니다!

원래 올려주신 html, css, js 파일들로 스킨을 적용하고 일부 css에서만 소량의 지식으로 손을 댔었는데, 현재로서는 제가 pug쪽은 전혀 모르는 상태라 한번 공부해보고 말씀 주신대로 변경해보겠습니다.

좋은 하루 되세요!

**Reply by Nonlinear14** [reply]
안녕하세요! 저도 같은 글 열기 방식 때문에 댓글 남깁니다. 소스코드 수정 적용까지 하기엔 제 코딩 지식이 짧은터라() 귀찮더라도 html 수정을 하려고 하는데 이 방법은 어떻게 해야 할까요?

**Reply by pronist** [reply, marked as answer]
4.10.1 패치에서 다시 과거처럼 기존 창에서 열리도록 바꾸었습니다~
